### DISCUSSION TITLE
Adding Planned downtime to SLA reports

### DISCUSSION THREAD

**Discussion by thijsvanloef** [sni/thruk#1312] [General]
Hi! 

I've been testing with Thruk reporting and was wondering if there is an option to add "Planned maintenance" to the SLA report?

**Comment by sni** [comment]
sorry for the late reply, you can choose whether downtimes are considered as outage or not in the sla report options.

![image](https://github.com/sni/Thruk/assets/150844/916cde67-4969-4e53-bfb1-7be96f114fe9)

But there is no reliable way to know if a downtime is planned or not.
